### ISSUE TITLE
Images using an absolute path (both file or remote URL) don't work with HTML source

### ISSUE THREAD

**Issue by pulkit110** [bitcrowd/chromic_pdf#80]
When specifying the HTML content directly to the `print_to_pdf`, the rendered PDF doesn't contain the image. It works ok when specifying the image through base64. Going through the code, it seems that when in `:html` mode, there is no wait for the `Page.frameStoppedLoading` notification like it is for the `:url` mode.  https://github.com/bitcrowd/chromic_pdf/blob/29a23ac70ba5b4fb659bd28d4396e69b90ab949b/lib%2Fchromic_pdf%2Fpdf%2Fprotocols%2Fprint_to_pdf.ex#L13-L34

Reproduction: 

```ex
path = System.tmp_dir() |> Path.join("output.pdf")

result = ChromicPDF.Template.source_and_options(content: content()) |> ChromicPDF.print_to_pdf(output: path)
IO.inspect(result)

defp content() do
  :erlang.iolist_to_binary(["<img src=\"https://homepages.cae.wisc.edu/~ece533/images/peppers.png\">"])
end
```

**Comment by maltoe** [comment]
Hi @pulkit110,

thanks for reporting! Also thanks for the hint on the cause of the bug :) 

IIRC the problem here was that the `frameStoppedLoading` notification actually isn't sent after `setDocumentContent`, as the frame isn't loading anything (rather, the command directly manipulates the contained DOM). 

However, it seems Chrome always emits `Page.loadEventFired` and `Page.domContentEventFired`. I pushed a branch with an await statement on the former notification, could you try if that fixes your problem? Unfortunately, these are all sent asynchronously from the target process, so they don't necessarily come in the right order. For instance, for my simple integration test fixture html (without any images, etc.) it sends `Page.loadEventFired` before the response to `setDocumentContent`. So either we drop the `await_response` and only wait for `loadEventFired`, or find a way to await multiple messages in any order.

**Comment by maltoe** [comment]
By the way, if you'd like to take a look at the messages Chrome sends, [this is a good place](https://github.com/bitcrowd/chromic_pdf/blob/e548d1203a726bd5e5b6182304d3380dd2dca5a0/lib/chromic_pdf/pdf/browser/channel.ex#L51) to insert an `IO.inspect(msg)` line.

**Comment by pulkit110** [comment]
Hi @maltoe, thanks a lot for the quick reply. Yes, I confirm that the fix on 7bfa31b works, and the images are now visible with HTML content :+1:. 

And thanks for the amazing library. It has been very fast compared to any other PDF generation solutions and has been very easy to set up for deployments too.

**Comment by maltoe** [comment]
@pulkit110 thanks again! I'll craft up a proper PR with some tests and docs, but won't get to do that until after the holiday. Hope that is acceptable for you.

**Comment by pulkit110** [comment]
Sure, that works for me. Thanks again. I have switched to the URL solution in the meantime by generating a temporary file.

**Comment by maltoe** [comment]
@pulkit110 published fix as version 0.6.2

**Comment by pulkit110** [comment]
Thanks @maltoe
